.github/workflows/test.yml

```diff
@@ -95,3 +95,47 @@ jobs:
       - name: Build
         run: pnpm run build
         working-directory: web
+      # - name: Install Playwright
+      #   run: pnpm run e2e-install
+      #   working-directory: web
+      # - name: Run Playwright tests
+      #   run: pnpm run e2e
+      #   working-directory: web
+      #   env:
+      #     # This might depend on your test-runner/language binding
+      #     PLAYWRIGHT_TEST_BASE_URL: djmoxvh6xpra8.cloudfront.net/
+  playwright-tests:
+    # needs: [test-frontend]
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    # if: github.event.deployment_status.state == 'success'
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup pnpm
```

Comment:
we also need to get the branch name so we can use it below as ${{ steps.branch.outputs.name }}.test.granted.run'. You can do so by adding the `common-fate/branch-name` action to this workflow.

```
      - uses: common-fate/branch-name@v1.1.2
        name: Get branch name
        id: branch
        with:
          max-length: 32
```